dockstore-client/src/main/java/io/dockstore/client/cli/nested/ApiClientExtended.java

```diff
@@ -16,16 +17,55 @@
 
 public class ApiClientExtended extends ApiClient {
 
+    /**
+     *
+     * @param key
+     * @return
+     */
+    private Optional<MediaType> getMediaType(Optional<String> key) {
+        // if key happens to be null or we don't recognize the key then return null
```

Comment:
This comment is no longer correct now that you have Optional

Reply:
removed comment